pages/index.js

```diff
@@ -1,9 +1,145 @@
-import Spots from "./spots";
+import Image from "next/image";
+import landingpageImage from "../public/images/landingpage.jpg";
+import styled from "styled-components";
+import Link from "next/link";
+import parenthoodGif from "../public/images/parenthood.gif";
 
-export default function Home({ spots, setSpot }) {
+export default function Home() {
   return (
     <>
-      <Spots spots={spots} setSpot={setSpot} />
+      <Background>
+        <Image
+          src={landingpageImage}
```

Comment:
Instead of importing the image, you can simply add the path here.

```suggestion
          src="/images/landingpage.jpg"
```